railgun/internal/mgrutils/ingressclass.go

```diff
@@ -0,0 +1,6 @@
+package mgrutils
+
+import "github.com/kong/kubernetes-ingress-controller/pkg/annotations"
+
+// IngressClass is the current "kubernetes.io/ingress.class" configured for controllers
+var IngressClass = annotations.DefaultIngressClass
```

Comment:
I think that making this setting a part of the global state is a bad idea. The purpose of `manager.Config` is to avoid keeping runtime parameters (like this) in the global state.

Reply:
:+1:  c529b1cdea99c5b62c8850dfc4a12e7877556d80